CRM/Report/Form/Mailing/Detail.php

```diff
@@ -440,7 +440,7 @@ public function mailingList() {
     $mailing->query($query);
 
     while ($mailing->fetch()) {
-      $data[mysql_real_escape_string($mailing->name)] = $mailing->name;
+      $data[$mailing->escape($mailing->name)] = $mailing->name;
```

Comment:
IMO, it's better to use CRM_Core_DAO::escapeString instead of calling DB_DataObject->escape directly